spec/presenters/setup_presenter_spec.rb

```diff
@@ -5,7 +5,8 @@
   let(:presenter) do
     described_class.new(current_user: user,
                         user_fully_authenticated: false,
-                        user_opted_remember_device_cookie: true)
+                        user_opted_remember_device_cookie: true,
+                        opt_out_rem_me: false)
```

Comment:
can we update the tests in this file to cover the different values for `opt_out_rem_me` or whatever we name it?

Reply:
Just wanted to highlight this, would love unit test coverage of the presenters if possible

Reply:
sure.  i thought this had to do with the name change...just reread

Reply:
ok.  check it out...